tensorflow/go/saved_model.go

```diff
@@ -45,7 +45,7 @@ type SavedModel struct {
 // exported in other languages, such as using tf.saved_model.builder in Python.
 // See:
 // https://www.tensorflow.org/code/tensorflow/python/saved_model/
-func LoadSavedModel(exportDir string, tags []string, options *SessionOptions) (*SavedModel, error) {
+func LoadSavedModel(exportDir string, tags []string, options *SessionOptions, runOptions []byte, metaGraph []byte) (*SavedModel, error) {
```

Comment:
I'm still a bit confused about why `metaGraph` is an argument to this function instead of a member of the returned `SavedModel` object.

Could you elaborate on that?

Also, do update the unittests so that they exercise these new options.